diff --git a/CMakeLists.txt b/CMakeLists.txt
index 4c2ef20..62e9391 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -117,6 +117,7 @@ target_include_directories(N64RecompElf PRIVATE
 
 target_link_libraries(N64RecompElf fmt)
 
+if (NOT ANDROID)
 # N64 recompiler executable
 project(N64RecompCLI)
 add_executable(N64RecompCLI)
@@ -166,6 +167,7 @@ target_sources(RecompModMerger PRIVATE
 )
 
 target_link_libraries(RecompModMerger N64Recomp)
+endif()
 
 # Live recompiler
 project(LiveRecomp)
@@ -194,6 +196,7 @@ target_include_directories(LiveRecomp PRIVATE
 
 target_link_libraries(LiveRecomp N64Recomp)
 
+if (NOT ANDROID)
 # Live recompiler test
 project(LiveRecompTest)
 add_executable(LiveRecompTest)
@@ -207,3 +210,4 @@ target_include_directories(LiveRecompTest PRIVATE
 )
 
 target_link_libraries(LiveRecompTest LiveRecomp)
+endif()
